src/components/ContentWidgets/Video/VideoEditor.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import './Video.css';
+
+/**
+ * Let the user create an embedded video by specifying 
```

Comment:
you can extend the entire comment to 100 characters before putting it on a newline.